EDU-14918 - RatingField (#1858)
vtexdocs/dev-portal-content: docs/faststore/components/molecules/rating-field.mdx

---
title: "RatingField"
components:
  - RatingField.tsx
---

> The `RatingField` component allows users to input ratings in their product reviews. It wraps an [Rating](https://developers.vtex.com/docs/guides/faststore/molecules-rating) and its corresponding [Label](https://developers.vtex.com/docs/guides/faststore/atoms-label).

Reviewer: ```suggestion
The `RatingField` component allows users to submit ratings when submitting product reviews. It wraps the [Rating](https://developers.vtex.com/docs/guides/faststore/molecules-rating) component, which displays the rating system, and the associated [Label](https://developers.vtex.com/docs/guides/faststore/atoms-label) component, which provides a text description for the rating field. 
```

## Overview

<Tabs items={['Example', 'Code']} defaultIndex="0">
  <Tab>
    <OverviewSection containerStyle={{ flexDirection:'column', alignItems: 'normal' }}>
      <RatingField label="Rate the product from 1 to 5 stars" id="rating-field-default" />
      <RatingField label="Rate the product from 1 to 5 stars" id="rating-field-error" error="This field is required." />
     </OverviewSection>
  </Tab>
  
  <Tab>
    ```tsx
      <RatingField label="Rate the product from 1 to 5 stars" id="rating-field-default" />
      <RatingField label="Rate the product from 1 to 5 stars" id="rating-field-error" error="This field is required." />
    ```
  </Tab>
</Tabs>

---

## Import

Import the component from [@faststore/ui](https://developers.vtex.com/docs/guides/faststore/components-index)

```tsx
import { RatingField } from '@faststore/ui'
```

### Import Styles into your FastStore project

To apply the styles of this component in your FastStore project, import the following into your stylesheet:

```scss
@import '@faststore/ui/src/components/molecules/RatingField/styles.scss';
```

> Follow the instructions in the [Importing FastStore UI component styles](https://developers.vtex.com/docs/guides/faststore/using-themes-importing-ui-components-styles) tutorial.

Reviewer: ```suggestion
For more information, refer to [Importing FastStore UI component styles](https://developers.vtex.com/docs/guides/faststore/using-themes-importing-ui-components-styles).
```
